docs/README.md

```diff
@@ -32,6 +32,7 @@ macOS (OS X) development environment.
 
 ![Web interface showing list of analysed projects and bugs](images/demo.gif)
 
+Check out our [DEMO](https://codechecker-demo.eastus.cloudapp.azure.com) showing some analysis results on open-source projects!
```

Comment:
Maybe we can highlight this a little bit more:
```suggestion
**:bulb: Check out our [DEMO](https://codechecker-demo.eastus.cloudapp.azure.com) showing some analysis results on open-source projects!**
```

This would look like this:
**:bulb: Check out our [DEMO](https://codechecker-demo.eastus.cloudapp.azure.com) showing some analysis results on open-source projects!**

Reply:
Maybe should be rephrase to:
...showing some analysis results **of** open-source projects!

Reply:
Right! Fixed 🔨